Options.py

```diff
@@ -916,6 +917,222 @@ class ItemSet(OptionSet):
     convert_name_groups = True
 
 
+class PlandoText(typing.NamedTuple):
+    at: str
+    text: str
```

Comment:
so i actually have a use case for this now with my implementation. the problem is that the data i need won't work as just a str unless i force users to put wacky specific formatting in the strings. i haven't looked into what it does but is there a way i can at least made `text` a `List[str]` without too much boilerplate?

Reply:
Doing so would probably require some manner of LttP refactor, but I'm not against the idea. LttP just goes with "users have to use specific formatting".